src/index.ts

```diff
@@ -1,3 +1,4 @@
+export { default as Theme } from "../lib/theme";
```

Comment:
would it be confusing to have a `Theme` object be exported here and also a `Theme` type be exported from `src/theme/global` ?

Reply:
Hmmmm i'm not sure. Should I call this `BaseTheme`?